traffic_ops/testing/api/v3/tc-fixtures.json

```diff
@@ -312,6 +312,7 @@
                 "initialDispersion": 1,
                 "logsEnabled": true,
                 "longDesc": "long desc",
+				"orgServerFqdn": "http://example.test",
```

Comment:
IDK if it's GH or something else, but this new line seems to have a different indentation than the others.

Reply:
No, it's actually different, because my text editor uses real indentation for indenting, whereas whoever made the original seems to have used spaces for indenting. I'll "fix" it to make it consistent.